src/index.jsx

```diff
@@ -1,22 +1,11 @@
-/* eslint-disable import/no-extraneous-dependencies */
-import React, { Fragment } from 'react';
+import React from 'react';
 import { render } from 'react-dom';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
-import SchemaTable from './components/SchemaTable';
-import theme from './theme';
+import SchemaViewer from './components/SchemaViewer';
 
 const root = document.getElementById('root');
 
 function App() {
-  return (
-    <Fragment>
-      <CssBaseline />
-      <ThemeProvider theme={theme}>
-        <SchemaTable />
-      </ThemeProvider>
-    </Fragment>
-  );
+  return <SchemaViewer />;
```

Comment:
It looks like we had everything setup properly for users to customize the theme.

Reply:
Yep, reverting back to the way things were setup originally!